Dockerfile.v3.2.x

```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:18.04
+HEALTHCHECK --start-period=15m CMD wget --quiet --tries=1 --no-check-certificate http://127.0.0.1:8088 || exit 1
```

Comment:
Just curious why a start period of 15m; that seems a bit excessive, no?  I was able to start and get to the point of receiving `Omada Controller started` in about 1 minute on an embedded box that has an `AMD G-T40E` which has a max clock speed of 1 GHz.  I would think that maybe 2-3 minutes; or 5 minutes if we are looking for a huge cushion might be good unless you're looking at a different use case.  Thoughts?